src/SFA.DAS.EmployerIncentives.Abstractions/DTOs/Queries/ClawbackStatusDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerIncentives.Abstractions.DTOs.Queries
+{
+    public class ClawbackStatusDto
+    {
+        public decimal? ClawbackAmount { get; set; }
+        public DateTime? ClawbackDate { get; set; }
```

Comment:
in what context are these going to be `null`?
also Clawback prefix is implied and not needed imho 

```suggestion
        public decimal Amount { get; set; }
        public DateTime Date { get; set; }
```

Reply:
The thinking was that this line
https://github.com/SkillsFundingAgency/das-employer-incentives/blob/298dc4949e9dcec18f31fa5674b6fa079cd4dd45/src/SFA.DAS.EmployerIncentives.Data/ApprenticeApplicationDataRepository.cs#L38
would return an empty model if no clawback found, but it returns null, so have changed the dto accordingly.